features/upgrade/api_auth/upgrade.feature

```diff
@@ -271,7 +271,7 @@ Feature: apiserver and auth related upgrade check
       | f | headless-services.yaml |
     Then the step should succeed
     Given I use the "service-ca-upgrade" project
-    And I wait for the "test-serving-cert" secret to appear
+    And I wait for the "test-serving-cert" secret to appear up to 120 seconds
```

Comment:
Default waiting time is 60s, we can see from https://github.com/openshift/verification-tests/blob/master/features/step_definitions/resources.rb#L114-L117, not sure if 120s is a appropriate value, let's go and see.

Reply:
yes, right. Thanks